src/test/unit_tests/test_nc_nsfs_cli.js

```diff
@@ -124,6 +124,39 @@ mocha.describe('manage_nsfs cli', function() {
             await exec_manage_cli(nc_nsfs_manage_entity_types.ACCOUNT, action, account_options2);
         });
 
+        mocha.it('cli bucket create - should fail bucket owner does not have allow_bucket_creation', async function() {
```

Comment:
worth adding an update flow tests to see we don't fail there

Reply:
What is the flow that you thought of?
- Since the account cannot create buckets, it should not have buckets to update (it cannot update a bucket that he doesn't own).
- I did a manual test (which is not just using the manage nsfs commands):
1. I created an account with `new_buckets_path`
2. I created a bucket
3. I manually changed the configuration file of the account (see my comment below): I removed the `new_buckets_path` and set the `allow_bucket_creation` to false.
4. I updated the bucket I already created (step 2) `--fs_backend GPFS` instead of the default successfully.

Currently, we only check the value of `allow_bucket_creation` only when the user tries to create a new bucket (both manage nsfs and bucketspace_fs).
1) I think that we should check it for every action that the user does on a bucket - add/update/delete and a user with `allow_bucket_creation` false will be able only to list and status (and from  bucketspace_fs to use GET ops).
Unfortunately, we cannot implement it now because in manage nsfs actions on a bucket are not related to a specific account (like we have the `requesting_account` in bucketspace_fs).

2) Today the value of `allow_bucket_creation` is inferred from the flag `new_buckets_path` (if we have it then it is true, and vice versa), but we do not have the option to unset `new_buckets_path` (like we have for `fs_backend` or `bucket_policy`).

Reply:
Summarizing:
1. The administrator is the role that uses manage nsfs, hence we throw an error only when trying to create a bucket for an account with `allow_bucket_creation` false. The rest of the action (delete, update, etc.) can be performed on the bucket since the administrator is the role that uses the system.
2. A unit test for the described scenario will be added after we can modify the `allow_bucket_creation` to false using manage_nsfs (either directly or by unsetting the value of `new_buckets_path`.